src/interface/swagger.json

```diff
@@ -1866,7 +1866,7 @@
             "$ref": "#/definitions/Extension"
           },
           "minItems": 1,
-          "maxItems": 16,
+          "maxItems": 17,
```

Comment:
Hey @shashi165 , @msk- , why is this being changed from 16? Did I miss some context here?